swift-sdk/IterableAPI.swift

```diff
@@ -498,6 +498,7 @@ public final class IterableAPI : NSObject {
     @objc public static var inAppManager: IterableInAppManagerProtocol {
         guard let internalImplementation = internalImplementation else {
             ITBError("IterableAPI is not initialized yet. InApp will not work now.")
+            assertionFailure("IterableAPI is not initialized yet. InApp will not work now.")
```

Comment:
`In-app`
Will this crash the app?

Reply:
Since this is assertionFailure() this will only crash in debug builds. It will be replaced with a noop in production builds.

Reply:
`InApp` -> `In-app`